tracer/resources/applications.py

```diff
@@ -116,7 +116,10 @@ def _append_application(default_attrs, specific_attrs={}):
 	@staticmethod
 	def _helper(app):
 		if app.type == Applications.TYPES["DAEMON"]:
-			return "service {0} restart".format(app.name)
+			if System.init_system() == "systemd":
```

Comment:
No problem to make it Arch specific, but is that really true? I used Fedora for years and all documentation used `systemctl`. My understanding was that `service` existed enitrely for legacy reasons